src/app/examples/list-example/list-example.component.html

```diff
@@ -1,24 +1,29 @@
 <kirby-list [items]="myItems" (itemSelect)="onItemSelect($event)">
-  <kirby-list-cell *kirbyListCell="let account" [lineOne]="account.title" [lineTwo]="account.detail"></kirby-list-cell>
-  <kirby-list-cell *kirbyListCell="let acount" [lineOne]="acount.subTitle" alignHorisontal="flex-end"></kirby-list-cell>
-  <kirby-list-cell *kirbyListCell="let acount" [lineOne]="acount.amount" alignVertical="center" alignHorisontal="flex-end"></kirby-list-cell>
+  <kirby-list-cell 
+    *kirbyListCell="let account" 
+    [lineOne]="account.title" 
+    [lineTwo]="account.detail">
+  </kirby-list-cell>
+  <kirby-list-cell *kirbyListCell="let account" [lineOne]="account.subTitle" alignHorisontal="flex-end"></kirby-list-cell>
```

Comment:
Maybe you could just format all the `kirby-list-cell` elements like the first?

Reply:
prettier to the rescue